### PR TITLE
Added deletemember feature

### PR DESCRIPTION
**Pull request by zylee348**
### Added deletemember feature and updated testcases.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddGroupMemberCommand.java [hunk 1]
```diff
@@ -65,11 +64,11 @@ public CommandResult execute(Model model) throws CommandException {
         try {
             groupToAddPerson = groupList.get(0);
         } catch (IndexOutOfBoundsException e) {
-            throw new CommandException(MESSAGE_INVALID_GROUP);
+            throw new CommandException(String.format(MESSAGE_INVALID_GROUP, this.personGroup));
         }
 
         if (groupToAddPerson.contains(personToGroup)) {
-            throw new CommandException(MESSAGE_DUPLICATE_PERSON_IN_GROUP);
+            throw new CommandException(String.format(MESSAGE_DUPLICATE_PERSON_IN_GROUP, this.name, this.personGroup));
```
Comment (Aloynz) [inline src/main/java/seedu/address/logic/commands/AddGroupMemberCommand.java:71]: good that the person's name and group are included in the description of the exception thrown

File: src/main/java/seedu/address/logic/commands/DeleteGroupMemberCommand.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+        Person editedPerson = new Person(
+                personToGroup.getName(), personToGroup.getPhone(), personToGroup.getEmail(),
+                personToGroup.getAddress(), personToGroup.getTags(), personToGroup.getAssignments(),
+                personGroupArrayList);
+
+        //deletes person from the group
+        Set<Person> groupMembers = new HashSet<>();
+        groupMembers.addAll(groupToDeletePerson.getMembers());
+        groupMembers.remove(editedPerson);
```
Comment (Aloynz) [inline src/main/java/seedu/address/logic/commands/DeleteGroupMemberCommand.java:85]: good use of HashSet to make the code for deletemember much more cleaner

**Review by Aloynz — COMMENTED** [review]
LGTM

File: docs/UserGuide.md [hunk 3]
```diff
@@ -7,13 +7,14 @@ TABS helps project team leaders **overview tasks assigned to group members**. It
 
 - [`Quick start`](#quick-start)
 - [`Features`](#features)
-  - Adding a person: [`addpeson`](#adding-a-person-adduser)
-  - Listing all persons: [`listpersons`](#listing-all-persons--list)
-  - Edit a person: [`editperson`](#editing-a-person--edituser)
-  - Locating persons by name: [`findperson`](#locating-persons-by-name-finduser)
-  - Deleting a person : [`deleteperson`](#deleting-a-person--deleteuser)
+  - Adding a person: [`addpeson`](#adding-a-person-addperson)
```
Comment (LianGuoYang) [inline docs/UserGuide.md:10]: Spelling error for `addpeson`. Can ignore it for now.

**Pull request merged**